### DISCUSSION TITLE
How to use imported svg icon or image in the background-image css in tailwind ???

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#9998] [Help]
Normally, we use various classname of tailwindcss for applying different CSS. But if we want to use `svg` file of an icon or an image as `background-image` css. How can we implemet that?
Like in below example, we are using `img` html element. How can we use this `svg` file as background-image so that we can insert text node as child nodes.
```
import React from 'react';
{/*images*/}
import ReactLogo from './logo.svg';

const App = () => {
  return (
    <div className="App">
      <img src={ReactLogo} alt="React Logo" />
    </div>
  );
}
export default App;
```

**Comment by wongjn** [comment]
You would use the `style` attribute, like:
```jsx
import React from 'react';
{/*images*/}
import ReactLogo from './logo.svg';

const App = () => {
  return (
    <div className="App">
      <div style={{ backgroundImage: ReactLogo }}>
        {/* … */}
      </div>
    </div>
  );
}
export default App;
```

You might need to do `ReactLogo.default` but it hopefully demonstrates the point.

**Reply by wongjn** [reply]
> Thanks @wongjn for your response. Above solution is inline style. I was expecting something we can do with tailwindcss. Anyways but I think we need to include `url()` inside the string literals like the below solution. Isn;t it ?
> 
> ```
>  <div className="App" style={{ backgroundImage: `url(${ReactLogo})`}} >
> ```[^1]

Seems like that could be it.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/9998#discussioncomment-4302358

**Comment by ProPhycient5** [comment]
Thanks @wongjn for your response.
Above solution is inline style. I was expecting something we can do with tailwindcss. Anyways but I think we need to include `url()` inside the string literals like the below solution.
Isn;t it ?

```
 <div className="App" style={{ backgroundImage: `url(${ReactLogo})`}} >
 ```

**Reply by wongjn** [reply]
Replied https://github.com/tailwindlabs/tailwindcss/discussions/9998#discussioncomment-4302505 to keep the reply chain together.
